src/Socket.c

```diff
@@ -99,6 +101,10 @@ int Socket_setnonblocking(int sock)
  */
 int Socket_error(char* aString, int sock)
 {
+#if defined(WIN32) || defined(WIN64)
+	int errno;
```

Comment:
This definition of a local variable `errno` is also needed to solve the problem!

Reply:
but this is needed, yes.  It was removed by the VxWorks commit